Source/ee/Vif1.h

```diff
@@ -21,7 +21,7 @@ class CVif1 : public CVif
 private:
 	void			ExecuteCommand(StreamType&, CODE) override;
 
-	void			Cmd_DIRECT(StreamType&, CODE);
+	void			Cmd_DIRECT(StreamType&, CODE&);
```

Comment:
Kinda pointless and possibly wasteful on 64-bits machines. `CODE` is a 4 bytes value, a reference is either 4 bytes or 8 bytes.